### PR TITLE
Adding Javascript 3 lecture notes

### PR DESCRIPTION
**Pull request by grod220**
So the next lecturers have resources and exercises to build off of

### CODE REVIEW

File: Week2/README.md [hunk 1]
```diff
@@ -15,10 +15,4 @@ Here are resources that we like you to read as a preparation for the second lect
 - Read about Asynchronous vs. Synchronous programming: http://www.hongkiat.com/blog/synchronous-asynchronous-javascript/
 - [Why closures are helpful with async code](http://stackoverflow.com/questions/13343340/calling-an-asynchronous-function-within-a-for-loop-in-javascript)
 
-### Fundamentals
```
Comment (mattiaslundberg) [inline Week2/README.md:18]: @grod220 was these removed by mistake? Or is there something I miss?

Comment (grod220) [reply Week2/README.md:18]: I'm pretty sure the links didn't work 🤔

Comment (mattiaslundberg) [reply Week2/README.md:18]: They work assuming you have forked the fundamentals repo (atleast that’s what I found)

Comment (mattiaslundberg) [reply Week2/README.md:18]: Added a PR to restore them, #5

**Pull request merged**